### DISCUSSION TITLE
I want to use Vercel for preview deployments only. How do I disable production one?

### DISCUSSION THREAD

**Discussion by 0x009922** [vercel/community#2899] [Help] [answered]
### Summary

The title is self-explanatory.

### Steps to Reproduce

N/A.

**Comment by smaeda-ks** [comment, marked as answer]
@0x009922 have you tried setting `git.deploymentEnabled` in your `vercel.json` file?
https://vercel.com/docs/concepts/projects/project-configuration/git-configuration#git.deploymentenabled
